### ISSUE TITLE
First row of sobol_sequence()

### ISSUE THREAD

**Issue by Paradoxeuh** [SALib/SALib#136]
Hi,

I'm trying to use your sample.sobol_sequence function but I've noticed that the first row is giving completely wrong values (like 2.13393829e+219). It seems that you are skipping first rows in your code when you use sobol_sequence() but I'm not sure to understand what is happening here.
Is it an expected behavior or am I doing something wrong?

Thank you for your work !

**Comment by willu47** [comment]
Hi @Paradoxeuh. Thanks for the question.  Please could you share some example code so that we can recreate this issue?

**Comment by Paradoxeuh** [comment]
Sure !

![capture du 2017-03-31 14-35-49](https://cloud.githubusercontent.com/assets/7406885/24550656/5c46a94a-161f-11e7-8e64-16b20d2243a8.png)

**Comment by Paradoxeuh** [comment]
Maybe this screenshot shows better my problem
![capture du 2017-03-31 14-44-25](https://cloud.githubusercontent.com/assets/7406885/24550903/955fa2c6-1620-11e7-9cf4-8fd8eb87209c.png)

**Comment by willu47** [comment]
Okay, thanks.  I'm looking into this now.

**Comment by willu47** [comment]
Okay, I confirm that I find the same issue.
```python
>>> from SALib.sample import sobol_sequence as ss
>>> ss.sample(100, 100)
array([[  0.00000000e+000,   1.26480805e-321,   0.00000000e+000, ...,
          3.43758150e+179,   6.01353875e-154,   4.18013514e-062],
       [  5.00000000e-001,   5.00000000e-001,   5.00000000e-001, ...,
          5.00000000e-001,   5.00000000e-001,   5.00000000e-001],
       [  7.50000000e-001,   2.50000000e-001,   2.50000000e-001, ...,
          2.50000000e-001,   2.50000000e-001,   7.50000000e-001],
       ..., 
       [  5.39062500e-001,   6.32812500e-001,   8.98437500e-001, ...,
          2.10937500e-001,   2.42187500e-001,   5.07812500e-001],
       [  7.89062500e-001,   3.82812500e-001,   1.48437500e-001, ...,
          9.60937500e-001,   9.92187500e-001,   7.57812500e-001],
       [  2.89062500e-001,   8.82812500e-001,   6.48437500e-001, ...,
          4.60937500e-001,   4.92187500e-001,   2.57812500e-001]])
```

**Comment by willu47** [comment]
It seems that the part of the code which is supposed to remove the first line of the array which contains these floating-point errors, isn't doing its job properly.

**Comment by willu47** [comment]
I've made a fix which you can install and try out using pip as follows:
```bash
pip install git+https://github.com/willu47/SALib@fix/sobol#egg=SALib
```

**Comment by jdherman** [comment]
Thanks @Paradoxeuh for noticing this. My mistake, I was creating the matrix using `np.empty` instead of `np.zeros` so there is junk in the first row that is never overwritten. Should be fixed in #138. We'll get this out to pypi in a patch today.

This will not affect any results generated using the `saltelli` sampling method, since that skips the first 1000 rows of the sequence regardless.
